main.go

```diff
@@ -151,6 +155,46 @@ func updateNDK(revision string) error {
 	return nil
 }
 
+func unWrapNDKDirIfNeeded(ndkHome string, revision string) error {
+	platformDirName := "platforms"
+	platformDir := filepath.Join(ndkHome, platformDirName)
+	p, err := pathutil.IsDirExists(platformDir)
+	if err != nil {
+		return fmt.Errorf("could not check if directory %s exists. Error: %v", platformDir, err)
```

Comment:
Usually we don't split a sentence with a `.`. In this case this sentence can be: `could not check if directory %s exists, error: %v`

Reply:
apply this comment to the upcoming occurrences also

Reply:
Fixed.